docs/reference/agent-config.txt

```diff
@@ -0,0 +1,45 @@
+.. _agent-configuration:
+
+Agent Configuration
+===================
+
+For both the master and the agent, configuration options may be set using a
```

Comment:
Is there a default path for the agent config file? And/or how do you specify the path to the config file when you start the agent? Can you specify an agent config file when using tools like `det-deploy`?

Reply:
There is no default file path but I can definitely add an example using `det-deploy`

Reply:
Update: `det-deploy` currently doesn't offer the ability to use a configuration file. I can give an example using `docker run` for passing a configuration file (which is how our agents are currently configured in environments where we need to add agent labels and other configuration).

Reply:
That sounds good to me. Would you mind filing a ticket for the det-deploy feature if there isn't one already?

Reply:
https://determinedai.atlassian.net/browse/DET-2986